core/domain/stats_jobs_one_off.py

```diff
@@ -34,6 +36,46 @@
 ])
 
 
+class RemoveBadPlaythroughsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Add one more job which runs over all models (whitelisted or not) and outputs the latest date of a playthrough that is affected (has None where it shouldn't be). (This should be verified against AUGUST_RELEASE_DATETIME before running the job.)

That job can also be used for post-validation (it should yield None if there are no problematic issues).

Make sure there are automated tests for this job.

Reply:
Done.